Sources/GRPCInteroperabilityTestsImplementation/InteroperabilityTestCases.swift

```diff
@@ -429,13 +429,20 @@ class ClientCompressedStreaming: InteroperabilityTest {
     let client = Grpc_Testing_TestServiceServiceClient(connection: connection)
 
     // Does the server support this test?
-    let probe = client.streamingInputCall()
+
+    // To find out we need to send an uncompressed probe. However we need to disable compression at
```

Comment:
nit: to find what out?

Reply:
Whether the server supports this test (line above) -- these lines should be joined.

Reply:
Right, sorry.